org.eclipse.scout.rt.rest.jersey.client/src/main/java/org/eclipse/scout/rt/rest/jersey/client/ScoutApacheConnector.java

```diff
@@ -855,4 +864,22 @@ public String getKey() {
       return "scout.rest.client.http.connectionTimeToLive";
     }
   }
+
+  public static class RestForceEncodeUriForRequestProperty extends AbstractBooleanConfigProperty {
+
+    @Override
+    public Boolean getDefaultValue() {
+      return true;
+    }
+
+    @Override
+    public String description() {
+      return "Specifies whether the URI should be re-encoded to force US-ASCII encoding (RFC 2396 compliant)";
```

Comment:
Indicate that the default value is true

Reply:
See https://github.com/eclipse-scout/scout.rt/compare/08d130fbec9b9edda3b767d7add7aaba7338cc73..731f385a5361637c93be984bc29be2cd0f405609